lib/media/drm_engine.js

```diff
@@ -617,23 +636,40 @@ shaka.media.DrmEngine = class {
    *
    * @return {!Promise}
    */
-  createOrLoad() {
-    // Create temp sessions.
+  async createOrLoad() {
```

Comment:
I noticed that the attach(video) function calls createOrLoad() but does not wait for the returned promise to resolve. Should it wait?

Reply:
I think it was done on purpose so that creating / loading sessions could be done in parallel of fetching manifest information, segments and more.